scripts/update_formula.php

```diff
@@ -1,54 +1,136 @@
 <?php
 
-$phpVersions = ['8.0', '8.1', '8.2', '8.3'];
+set_error_handler(function (int $errno, string $errstr, string $errfile, int $errline) {
+    if (!(error_reporting() & $errno)) {
```

Comment:
I'm not loving the idea that if a script throws and exception or not is dependent on the error_reporting configuration. 

As this is mainly (elusively?) watching file_put_contents warnings here it doesn't matter much but I'd rather not have this clause

Reply:
Dropped.